frontend/packages/knative-plugin/src/models.ts

```diff
@@ -252,3 +253,29 @@ export const EventingBrokerModel: K8sKind = {
   crd: true,
   color: knativeEventingColor.value,
 };
+
+export const KafkaModel: K8sKind = {
+  apiGroup: STRIMZI_KAFKA_APIGROUP,
+  apiVersion: 'v1beta1',
+  kind: 'Kafka',
+  label: 'Kafka',
+  labelPlural: 'Kafkas',
+  plural: 'kafkas',
+  id: 'kafka',
+  abbr: 'K',
+  namespaced: true,
+  crd: true,
```

Comment:
we can add `color: knativeEventingColor.value,` as well

Reply:
We are not using color here so I didn't add it. Will add it if it adds any value